packages/transition-frontend/webpack.config.js

```diff
@@ -135,7 +153,7 @@ module.exports = (env) => {
           'HOST'                        : JSON.stringify(process.env.HOST),
           'TRROUTING_HOST'              : JSON.stringify(process.env.TRROUTING_HOST),
           'PROJECT_SOURCE'              : JSON.stringify(process.env.PROJECT_SOURCE),
-          'NODE_ENV'                    : JSON.stringify(process.env.NODE_ENV),
+          'process.env.NODE_ENV'                    : JSON.stringify(process.env.NODE_ENV),
```

Comment:
Is this change right?

Reply:
It fixed the `Conflicting values` warning I was getting before. Not sure where I found the suggestion but <https://github.com/nrwl/nx/issues/7924#issuecomment-1043143337> mentions the same warning.

Reply:
iiuc, ceci va setter la variable process.env.process.env.NODE_ENV. La source de la conflicting values warning devait être ailleurs?

Reply:
J'ai passé beaucoup (trop) de temps à chercher pour résoudre chaque problème de la migration webpack. Je n'ai malheureusement plus de gaz pour continuer cette épopée.

Reply:
> J'ai passé beaucoup (trop) de temps à chercher pour résoudre chaque problème de la migration webpack. Je n'ai malheureusement plus de gaz pour continuer cette épopée.

Haha, c'est bon merci, tu t'es rendu plus loin que moi avant d'abandonner cette épopée! On pourra reprendre de là.